operator/cmd/main.go

```diff
@@ -90,13 +92,17 @@ func main() {
 		"The address the static file server binds to.")
 	flag.StringVar(&programFSAdvAddr, "program-fs-adv-addr", envOrDefault("PROGRAM_FS_ADV_ADDR", ""),
 		"The advertised address of the static file server.")
+
+	// Configure Zap-based logging for klog/v2 and for the controller manager.
+	// Write to stdout by default.
 	opts := zap.Options{
-		Development: true,
```

Comment:
This `Development` flag is equivalent to debug-level + console format. I think it simplest to simply remove this line in favor of being explciit in Helm, quickstart, and launch.json.